src/components/CompareResults/ResultsTable.tsx

```diff
@@ -60,13 +59,7 @@ function processResults(results: CompareResultsItem[]) {
 const allRevisionsOption =
   Strings.components.comparisonRevisionDropdown.allRevisions.key;
 
-const customStyles = {
-  boxShadow: 'none',
```

Comment:
instead of overrinding boxShadow I simply used a Box (which is really like a div expect it's more integrated in mui... we could probably use a div here actually, not sure).

I tried `<Paper elevation={0}` but this also changed the background in dark mode, so this wasn't working as expected.